doc/design/error-handling.md

```diff
@@ -240,84 +234,84 @@ if(result.has_error()) {
 Alternatively a functional approach can be used.
 
 ```
-auto successFunc = [](SomeType& value) { 
+auto successFunc = [](SomeType& value) {
     //proceed by using the value
 };
 
-auto errorFunc = [](Error& error) { 
+auto errorFunc = [](Error& error) {
     //handle the error
 };
 
 func(arg).and_then(successFunc).or_else(errorFunc);
 ```
 
-# Open Points
+### Open Points
 
-## Centralized Error Handling
+#### Centralized Error Handling
 
 It may be desirable to have a centralized error handling instance where runtime errors on application side are logged and (maybe) handled.
 This could also be done in RouDi (by sending information to RouDi), but RouDi already has too much responsibility. Preferably this should be done by a separate application with this sole purpose.
 If the application cannot reach the central handler, it shall try to handle the error locally if possible (at least log it).
 
 However, it might be too slow if this would rely on error transmission and responses. If this is to be implemented, the exact mechanism has to be decided on.
 
-## 3rd Party Error Handling
+#### 3rd Party Error Handling
 We need to decide how to provide an interface for 3rd party error handling, especially for the runtime. This interface will probably rely on hooks/callbacks. The signature and callsites of these need to be discussed.
 This is related to centralized error handling as well.
 
-## Overriding Specific Error Reaction
+### Overriding Specific Error Reaction
```

Comment:
I think this a subchapter of "Open Points". Please add one ``#``.

Reply:
Fixed